### PR TITLE
Double application of softmax

### PR DESCRIPTION
**Pull request by plooney**
I think calling the softmax on the last layer and then using 

loss = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(net, Y))

results in two applications of softmax

### CODE REVIEW

**Comment by aymericdamien** [comment]
Hi, that is right! Thanks for fixing that.
